tools/sdk/units.ts

```diff
@@ -68,3 +73,7 @@ export function getMintDecimalAmountFromNatural(
 ) {
   return new BigNumber(naturalAmount.toString()).shiftedBy(-mint.decimals)
 }
+
+export function numberWithCommas(num) {
```

Comment:
This should be locale specific, we can't assume commas as separators. See `fmtMintAmount` as an example